src/bespokelabs/curator/request_processor/openai_batch_request_processor.py

```diff
@@ -22,6 +22,7 @@
 logger = logging.getLogger(__name__)
 
 MAX_REQUESTS_PER_BATCH = 50_000
+MAX_BYTES_PER_BATCH = 200_000_000
```

Comment:
https://platform.openai.com/docs/guides/batch#:~:text=Per%2Dbatch%20limits%3A%20A%20single,to%20200%20MB%20in%20size.
200MB is 200 * 1024 * 1024, add this link as a comment here for reference